ajax/libs/OrgChart.js/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "OrgChart.js",
+  "description": "organization chart plugin based on ES6",
+  "filename": "OrgChart.min.js",
+  "keywords": [
+    "es6",
+    "ES2015",
+    "organization chart",
+    "tree-like"
+  ],
+  "author": {
+    "name": "dabeng",
+    "email": "dabeng413@gmail.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dabeng/OrgChart.js.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/dabeng/OrgChart.js.git",
+    "fileMap": [
+      {
+        "basePath": "lib",
+        "files": [
+          "+(orgchart|OrgChart)*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "orgchart*"
```

Comment:
Hi @sashberd, I think we only need files in `lib`, right? Any ref to point out we need files in `src`?
Because this auto-update config will fetch files in `src`.